src/test/java/com/hocs/test/pages/foi/FOIDispatch.java

```diff
@@ -26,4 +30,32 @@ public void selectResponse(String responseType) {
     public void selectOutcomeOfTheCase(String outcome) {
         recordCaseData.selectSpecificRadioButtonFromGroupWithHeading( outcome,"What was the outcome of this case?");
     }
+
+    public void selectRandomExemptionOrException() {
+        List<WebElementFacade> exemptionsList =
+                findAll("//div[@class='govuk-checkboxes__item']");
+        recordCaseData.checkRandomCheckboxFromList(exemptionsList);
+    }
+
+    //To-DO
+    public void selectRandomExceptionAndExemption() {
+        List<WebElementFacade> exemptionsList =
+                findAll("//div[@class='govuk-checkboxes__item']");
+        recordCaseData.checkRandomCheckboxFromList(exemptionsList);
+//        recordCaseData.checkRandomCheckboxFromList(exemptionsList);
+    }
+
+    public void selectNonDispatchOutcomeOfTheCase() {
```

Comment:
Im not overly familiar with FOI workflow, but is this a situation where we could happily press any radio button under the heading "What was the outcome of this case" ?
If thats so we could use recordCaseData,selectRandomRadioButtonFromGroupWithHeading, and then we wouldnt need to create the list manually.
If only some of the radio buttons are acceptable to press, and some others would cause a failure, then feel free to close this conversation.

Reply:
We can select any random checkbox, will create the selectRandomCheckboxButtonFromGroupWithHeading